utils/rules_processor.go

```diff
@@ -111,14 +111,34 @@ func (r *RulesProcessor) ProcessLabelRules(e interface{}) {
 				columns := r.gh.ListProjectColumns(*projID)
 				if colID, ok := r.gh.GetCardColumnIDByName(columns, rule.Column); ok {
 					//the value exists
-					r.gh.CreatetProjectCard(rule.Content, *e.PullRequest.ID, colID)
+					r.gh.CreateProjectCard(rule.Content, *e.PullRequest.ID, colID)
 				} else {
 					log.Print("Unable to get Column ID")
 				}
 			}
 		}
-
 	case *github.IssuesEvent:
 		log.Print("received an Issue to process label rules", e)
+		if *e.Action != "labeled" && *e.Action != "unlabeled" {
+			log.Println(*e.Action)
```

Comment:
may want to add context to this print, e.g. `log.Println("Ignoring issue action", *e.Action)`